### ISSUE TITLE
Bitmap Font Tools

### ISSUE THREAD

**Issue by mattdesl** [mattdesl/ideas#5]
NodeJS tools built on Freetype2 and/or Opentype.js to replace [gdx-fontpack](https://github.com/mattdesl/gdx-fontpack). Vector SVG output would also be nice.

Also a tool that can take in vector paths or bitmasks and produce SDF images for scalable rendering.

**Comment by ayamflow** [comment]
I'd love to be able to include the font pipeline in our build process. Download, subset, convert (to css, 3d,   bmp...)... It seems there is a lack of CLI/API for this kind of tasks.

**Comment by mattdesl** [comment]
@ayamflow yeah the automation for fonts is generally lacking compared to npm/js/etc.

some updates on this issue:
- [node-fontnik](https://github.com/mapbox/node-fontnik) can render a font to a SDF tile (for GPU rendering)
- [sdf-bitmap-glyphs](https://www.npmjs.com/package/sdf-bitmap-glyphs) is a simple high-level wrapper on `node-fontnik`
- [ndarray-bin-pack](https://github.com/Jam3/ndarray-bin-pack/blob/master/demo/index.js) can be used to pack the tiles into an atlas

Still a few more things before it can fully replace `gdx-fontpack` (https://github.com/mapbox/node-fontnik/issues/90) but getting there.

**Comment by ayamflow** [comment]
Also saw this one: [ttf2woff2](https://github.com/nfroidure/ttf2woff2), convenient since ttf + woff 1/2 are enough for evergreen browsers. But it's more for the web side (not bitmap).
